meta-iot2000-example/recipes-node/node-red-contrib-boolean-logic/node-red-contrib-boolean-logic_0.0.3.bb

```diff
@@ -0,0 +1,18 @@
+# Recipe created by recipetool
+
+SUMMARY = "A set of Node-RED nodes for boolean logic"
+# WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
```

Comment:
It actually is MIT according to https://github.com/PerMalmberg/node-red-contrib-boolean-logic, so the comment may be removed.